astronomer/providers/databricks/hooks/databricks.py

```diff
@@ -50,6 +50,26 @@ async def get_run_state_async(self, run_id: str) -> RunState:
 
         return RunState(life_cycle_state, result_state, state_message)
 
+    async def get_run_response(self, run_id: str) -> Dict[str, Any]:
```

Comment:
`get_run_response` and `get_run output response` do the same task if we even send the endpoint as an argument. WDYT?

Reply:
yes, we can do that